zynqmp/optee-client/optee-client.bb

```diff
@@ -0,0 +1,57 @@
+OPTEE_VERSION ??= "latest"
+SRCREV ??= "${AUTOREV}"
+BRANCH ??= "master"
+
+DESCRIPTION = "OP-TEE Client"
+
+#defined as closed license to prevent MD5 checksum verification
+LICENSE = "CLOSED"
```

Comment:
this is false.
try something like this:
```
LICENSE = "BSD"
LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=69663ab153298557a59c67a60a743e5b"
```

Reply:
Agree with you but license file of one of the three repositories has changed since version 3.4.0. therefore to switch between the latest version of OP-TEE and an older one, one would have to update the md5 hash in BB recipe. This is quite annoying. As far as I know, the only way to disable license checksum verification is to set license variable to "closed". If this is a problem, I believe that the only solution is to remove the possibility to build a given version of OP-TEE.

Reply:
optee-client and optee-test `LICENSE` files are quite stable, only optee-os recently changed. Maybe use this `CLOSED` hack only for optee-os, with an explicit comment:
```
# Define as closed license to prevent MD5 checksum verification since
# LICENSE file changed around 3.5.0 make this recipe less flexible.
# LICENSE = "BSD"
# ante 3.5.0: LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=69663ab153298557a59c67a60a743e5b"
# post 3.5.0: LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=c1f21c4f72f372ef38a5a4aee55ec173"
LICENSE = "CLOSED"
```